src/Digdir.Domain.Dialogporten.Application/Features/V1/ServiceOwner/Dialogs/Queries/List/ListDialogQuery.cs

```diff
@@ -1,22 +1,57 @@
 ﻿using AutoMapper;
 using AutoMapper.QueryableExtensions;
+using Digdir.Domain.Dialogporten.Application.Common;
+using Digdir.Domain.Dialogporten.Application.Common.Extensions;
+using Digdir.Domain.Dialogporten.Application.Common.Extensions.Enumerable;
 using Digdir.Domain.Dialogporten.Application.Common.Pagination;
 using Digdir.Domain.Dialogporten.Application.Common.Pagination.OrderOption;
 using Digdir.Domain.Dialogporten.Application.Common.ReturnTypes;
 using Digdir.Domain.Dialogporten.Application.Externals;
+using Digdir.Domain.Dialogporten.Domain.Dialogs.Entities;
+using Digdir.Domain.Dialogporten.Domain.Localizations;
 using MediatR;
 using OneOf;
+using static Digdir.Domain.Dialogporten.Application.Common.Expressions;
 
 namespace Digdir.Domain.Dialogporten.Application.Features.V1.ServiceOwner.Dialogs.Queries.List;
 
-public sealed class ListDialogQuery : PaginationParameter<ListDialogQueryOrderDefinition, ListDialogDto>, IRequest<ListDialogResult> { }
+public sealed class ListDialogQuery : SortablePaginationParameter<ListDialogQueryOrderDefinition, ListDialogDto>, IRequest<ListDialogResult>
```

Comment:
Skal tjenesteeier ha mulighet til å sortere? Kanskje ikke dumt 🤔

Reply:
Dunno! 
Sto `Ha støtte for paginering, sortering og limit tilsvarende søke-API-et for sluttbrukere` i issuet i hvert fall 😅